src/client/features/enrichment-map/cy/index.js

```diff
@@ -30,11 +30,19 @@ let bindEvents = cy => {
     let node = evt.target;
     let getEnrichmentTooltipData = () => {
       let id = node.data('id');
-      if(id.includes('GO')) return ServerAPI.getGoInformation(id);
-      else if(id.includes('REAC')) return ServerAPI.getReactomeInformation(id);
+      if(id.includes('GO')) return ServerAPI.getGoInformation( id.replace("GO:", "") );
```

Comment:
Safer if these were tests against a regex. I can't find any solid db id information for GO/Reactome, but seem to be similar to `/^GO:\d+$/` etc.